### PR TITLE
Fixed test

### PR DESCRIPTION
**Pull request by 9renpoto**
- Added prettier format
- update dependencies
- fixed tests

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -9,4 +9,5 @@
 [lints]
 
 [options]
-unsafe.enable_getters_and_setters=true
```
Comment (9renpoto) [inline .flowconfig:12]: deprecated https://github.com/facebook/flow/issues/5594#issuecomment-354560304

**Comment by matheusmr13** [comment]
Hello there! (:
Thanks for your PR. Any questions you may have about this project (even that I'm not actively working on this), I'm here to answer.

I'm curious about this interest, are u using this lib in some project of yours? If so, we can work together to make this project updated and with all features that u may need.

I've created a new branch to merge PRs, can you change this merge to develop?
Thanks!

**Comment by Surgo** [comment]
@matheusmr13 Thank you for your cool lib.
We'll use this lib on my production.

File: test/rootLevel.hook.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { before } from 'mocha';
+import extendGlobal from '../src';
+
+before(() => extendGlobal(global));
```
Comment (9renpoto) [inline test/rootLevel.hook.js:4]: https://mochajs.org/#root-level-hooks

**Pull request closed without merging**